kong/db/dao/plugins/go.lua

```diff
@@ -1,6 +1,7 @@
 local ffi = require("ffi")
 local cjson = require("cjson.safe")
 local ngx_ssl = require("ngx.ssl")
+local ngx_pipe = require "ngx.pipe"
```

Comment:
Didn't get to the bottom of the issue, but moving `ngx.pipe` to the scope where it's used in fixes the tests; further, the error only happens if certain combinations of specs are executed in a specific order (presumably some side effect going on in those specific paths). Can look further into this later on, but for now I'd propose we simply move the `require` : )

Reply:
thanks!  i saw that the failure was clearly a test thing.... will experiment on the best place to put it